pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -1152,6 +1152,8 @@ spec:
             - --size
             - {{ .Volume.capacity }}
             - /openebs
+            securityContext:
```

Comment:
Should this be done via a condition?
e.g. We can have a CASConfig which if enabled can set this up
```yaml
name: PriviledgedReplica
enabled: "true"
```
cc @vishnuitta @kmova